packages/utils/index.js

```diff
@@ -1,26 +1,22 @@
 // @flow
 
 // Helpers
-export { capitalize } from './helpers/capitalize';
+export { Capitalize } from './helpers/Capitalize';
```

Comment:
Thanks, looks definitely better. I'm just not that sure about those few formatter functions exported with capital letter. 
For me it looks more like class or constant at first glance.
And I'll be more for keeping it with lower case, like in case of `withLayoutContext` (but keep filename with capital).
What do you think?

Reply:
Yep, It also looks weird to me. Will change it now.